code/game/machinery/computer/atmos_alert.dm

```diff
@@ -14,7 +14,7 @@ var/global/list/minor_air_alarms = list()
 
 /obj/machinery/computer/atmos_alert/Initialize()
 	. = ..()
-	atmosphere_alarm.register_alarm(src, /obj/machinery/computer/station_alert/update_icon)
+	atmosphere_alarm.register_alarm(src, /atom/movable/.proc/update_icon)
```

Comment:
why the dot before proc? Is this valid code?

Reply:
Correctly used because we have .proc/update_icon.